src/providers/virtual/qgsvirtuallayerprovider.h

```diff
@@ -54,6 +54,18 @@ class QgsVirtualLayerProvider: public QgsVectorDataProvider
     QgsAttributeList pkAttributeIndexes() const override;
     QSet<QgsMapLayerDependency> dependencies() const override;
 
+    /**
+     * Interrupts the pending query.
+     * \since QGIS 3.2
+     */
+    bool cancel() override;
```

Comment:
Better not to repeat docstrings for overridden methods, unless they add extra information for a particular subclasses' implementation. Doxygen automatically copies the base class docs, and by overridding them here you risk missing out on any future documentation improvements added to the base class.

Reply:
OK, I didn't know that. I removed the documentation for these methods.